kaldb/src/main/java/com/slack/kaldb/preprocessor/PreprocessorService.java

```diff
@@ -195,8 +194,9 @@ protected static Topology buildTopology(
     ValueMapper<byte[], Iterable<Trace.Span>> valueMapper =
         PreprocessorValueMapper.byteArrayToTraceSpans(dataTransformer);
 
+    int lingerMs = Integer.parseInt(kafkaProperties.getProperty("producer.linger.ms", "100"));
```

Comment:
I wouldn't use lingerMs as the config for the sticky partitioner - I would have this as an entirely separate kaldb config option if we decide to use this approach.